### PR TITLE
Added duration_to_bytes to FileMockMicrophone

### PR DESCRIPTION
**Pull request by TheRealDGD**
## Description
Added duration_to_bytes to fix AttributeError: 'FileMockMicrophone' object has no attribute 'duration_to_bytes'
in call from mycroft-core/mycroft/client/speech/mic.py _wait_until_wake_word

## How to test
./start-mycroft.sh wakewordtest

## Contributor license agreement signed?
CLA [X] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

### CODE REVIEW

**Comment by devs-mycroft** [comment]
Hello, @TheRealDGD, thank you for helping with the Mycroft project! We welcome everyone
into the community and greatly appreciate your help as we work to build an AI
for Everyone.

To protect yourself, the project, and users of Mycroft technologies we require
a Contributor Licensing Agreement (CLA) before accepting any code
contribution. This agreement makes it crystal clear that along with your
code you are offering a license to use it within the confines of this project.
You retain ownership of the code, this is just a license.

Please visit https://mycroft.ai/cla to initiate this one-time signing. Thank
you!

**Comment by TheRealDGD** [comment]
Hello,

I already signed the CLA.
Are there additional steps needed after signing?

mycroft-developers ***@***.***> schrieb am Sa. 12. Feb. 2022
um 12:18:

**Comment by forslund** [comment]
Hi, the CLA has a manual step where your git username is added to https://github.com/MycroftAI/contributors, @krisgesling usually handles this. If you signed the CLA recently give it a couple of days. If it was a bit ago hopefully gez will see the mention here or we'll ping him again.

File: test/wake_word/wake_word_test.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def unmute(self):
     def close(self):
         self.stream.close()
 
+    def duration_to_bytes(self, ww_duration):
+        return int(ww_duration*self.SAMPLE_RATE*self.SAMPLE_WIDTH)
```
Comment (forslund) [inline test/wake_word/wake_word_test.py]: Maybe add spaces around the `*` to match the general code style

There is a blank line missing here as well. (There should be 2 blank lines between classes / global functions, 1 blank line between methods)

**Review by forslund — COMMENTED** [review]
Thanks for fixing this issue, the code looks like it does what it should do.

I commented on a couple of small code style issues in the code, very minor but I think one of them will make the code style checker angry.

After that I think this can be merged.

**Comment by krisgesling** [comment]
CLA received - thanks!

Just FYI - the tests are borked from a Python 3.6 issue unrelated to this PR. Hopefully will have that sorted today.

**Review by forslund — DISMISSED** [review]
Looks nice now!

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/core/PR-3071))

**Review by krisgesling — APPROVED** [review]

**Comment by krisgesling** [comment]
Excellent, thanks DGD!

**Pull request merged**